homeassistant/components/here_travel_time/sensor.py

```diff
@@ -216,55 +225,32 @@ def _are_valid_client_credentials(here_client: herepy.RoutingApi) -> bool:
     return True
 
 
-class HERETravelTimeSensor(SensorEntity):
+class HERETravelTimeSensor(SensorEntity, CoordinatorEntity):
     """Representation of a HERE travel time sensor."""
 
     def __init__(
         self,
         name: str,
-        origin: str,
-        destination: str,
-        origin_entity_id: str,
-        destination_entity_id: str,
-        here_data: HERETravelTimeData,
+        traffic_mode: str,
```

Comment:
Is this really a string? It looks like a bool in the configuration documentation and is treated like a bool below.

Reply:
Thanks I missed that!